examples/code/imgui_example/imgui_example.cpp

```diff
@@ -42,6 +42,92 @@ namespace
     u32                s_raster_state_cull_back = 0;
     u32                s_default_depth_stencil_state = 0;
 
+    ImGuiStyle &custom_theme(){
```

Comment:
is the the old `dev_ui` default theme values? I would like to keep these colours as the default from pmtech vs the default `ImGui` values, just as I have become attached to them over time :)

This could move to dev_ui, as `pmtech_imgui_theme` or something.

Reply:
Oh this is a different one which might not be good if you plan on keeping it xD I can alter it as well.

Reply:
I can definitely make it a little more well rounded as well.